app/src/main/res/layout/layout_footer_weather.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="horizontal"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```

Comment:
Bỏ LinearLayout, dùng constraint cho các view luôn
Nếu k hiểu phần nào về Constraint thì **To** lên box để hỏi cho nhanh